source/gems/dry-types/index.html.md

```diff
@@ -16,7 +16,80 @@ sections:
   - enum
 ---
 
-`dry-types` is a simple and extendable type system for Ruby; useful for value coercions, applying constraints, defining complex structs or value objects and more. It was created as a virtus' successor.
+`dry-types` is a simple and extendable type system for Ruby; useful for value coercions, applying constraints, defining complex structs or value objects and more. It was created as a successor to [Virtus](https://github.com/solnic/virtus).
+
+### Example usage
+
+```ruby
+require 'dry-types'
+
+module Types
+  include Dry::Types.module
+end
+
+Types::Strict::String["foo"]
+# => "foo"
+Types::Strict::String["10000"]
+# => "10000"
+Types::Coercible::String[10000]
+# => "10000"
+Types::Strict::String[10000]
+# Dry::Types::ConstraintError: 1000 violates constraints
+Types::Coercible::Int[1234]
+# => 1234
+Types::Coercible::Int["1234"]
+# => 1234
+Types::Coercible::Int[1234.5]
+# => 1234
+Types::Strict::Int[1234.5]
+# Dry::Types::ConstraintError: 1234.5 violates constraints
+Types::Strict::Float[1234.5]
+# => 1234.5
+Types::Bool[true]
+# => true
+Types::Bool[false]
+# => false
+Types::Bool["not a bool"]
+# Dry::Types::ConstraintError: "not a bool" violates constraints
+```
+
+See [Built-In Types](/gems/dry-types/built-in-types/) for a full list of types.
+
+You can use dry-types with [dry-struct](http://dry-rb.org/gems/dry-struct)
+to create a class whose attributes have clearly-defined, coercible types:
```

Comment:
Maybe just "to create a class with typed attributes"? They don't have to be coercible, or even strict, it depends on the use cases.